src/OrchardCore.Cms.Web/appsettings.json

```
@@ -235,6 +237,9 @@
     //},
     //"OrchardCore_Localization_CultureOptions": {
     //  "IgnoreSystemSettings": true
-    //}
+    //},
+    "OrchardCore_Tenants": {
+      "TenantRemovalAllowed": true // Whether tenant removal is allowed, false by default.
```

```suggestion
      "TenantRemovalAllowed": false // Whether tenant removal is allowed or not. Default is false.
```
Also be sure to comment out this section